File: app/src/main/java/com/jh/societymember/MainActivity.java
package com.jh.societymember;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.google.firebase.auth.FirebaseAuth;

public class MainActivity extends AppCompatActivity {

    private FirebaseAuth mFirebaseAuth;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn_logout =findViewById(R.id.btn_logout);

        btn_logout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //로그아웃하기
//                mFirebaseAuth.signOut(); -> 이게 있어서 파괴가 안됬었음! 이유 알아보기!
                Intent intent= new Intent(MainActivity.this, LoginActivity.class);
                startActivity(intent);
                finish();
            }
        });
       // mFirebaseAuth.getCurrentUser().delete(); //탈퇴 처리!

        Button btn_TInsert =findViewById(R.id.btn_TInsert);
        btn_TInsert.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent2= new Intent(MainActivity.this, TInsertActivity.class);
                startActivity(intent2);
            }
        });// end btn_TInsert

        Button btn_TMInsert =findViewById(R.id.btn_TMInsert);
        btn_TMInsert.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent3= new Intent(MainActivity.this, TMInsertActivity.class);
                startActivity(intent3);
            }
        });// end btn_TMInsert

        Button btn_TRead =findViewById(R.id.btn_TRead);
        btn_TRead.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent4= new Intent(MainActivity.this, TReadActivity.class);
                startActivity(intent4);
            }
        });// end btn_TRead

        Button btn_TUpdate =findViewById(R.id.btn_TUpdate);
        btn_TUpdate.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent= new Intent(MainActivity.this, TUpdateActivity.class);
            }
        });// end btn_TUpdate

    }
}

File: practice3/hongdroid12_logkt/src/main/java/com/jh/hongdroid12_logkt/MainActivity.kt
package com.jh.hongdroid12_logkt

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    var a: Int =0;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        a=2

        if(a==2){
            Log.e("if 문", "Enter") //Log.d는 밑에 로그캣에서 형식을 debug로 한 것! d이기 때문!
        }else if(a==1){
            Log.d("if 문", "Enter2") //Log.d는 밑에 로그캣에서 형식을 debug로 한 것! d이기 때문!
        }

    }
}